CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and
 this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
There are some user-visible changes that aren't documented in the CHANGELOG. Specifically, the APIs that previously allowed pagination within a range between `after` and `before`, e.g., `clusters`, `eventList`, and `models`, now only accept one of these parameters. I'm concerned that this might be a regression, but if dropping this feature was intentional, it would be great to document it in the CHANGELOG, since it affects web clients. Could you please add a note about this change?

Reply:
As of now, it looks good enough that all APIs just provide this restricted combination of parameters, and it helps maintaining consistency. So, I added a note.